updating/understanding_updates/intro-to-updates.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Introduction to OpenShift updates
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update an {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators are automatically notified when an update is available for their cluster.
+
+The OpenShift Update Service (OSUS) builds a graph of update possibilities based on release images in the registry.
+The graph is based on recommended, tested update paths from a specific version. {product-title} clusters connect to the Red Hat Hybrid Cloud servers and identify which clusters the user is running, along with the version information. OSUS responds with information about known update targets.
+
+An update begins when either a cluster administrator or an automatic update controller edits the custom resource (CR) of the Cluster Version Operator (CVO) with a new version to update to.
```

Comment:
> An update begins when either a cluster administrator or an automatic update controller edits the custom resource (CR) 

Generally a cluster administrator will probably not edit the CR directly. And update does not actually begin because we have precondition checks in the beginning. I'd personally prefer being vague to say,

An update is triggered when the custom resource (CR) of the Cluster Version Operator (CVO) is edited with a new version to update to.

Reply:
correct. setting the target in CR (spec.desiredUpdate) does not guarantee an upgrade being triggered. it can either reference an invalid target, or fail a precondition check.  
as well since we even decided to omit `--to-image`, we may even more so desire to omit tinkering with CR directly, which is even more low level

Reply:
The doc is supposed to explain what is happening at the background, its purpose is to give the reader insights into the low level, including how (mostly the usual path) things are reflected in the CR.

The doc also considers the download and checks to be part of the upgrade. Setting spec.desiredUpdate triggers the whole process, it can just fail very early.

Reply:
Unless there is strong opposition, I propose to keep this as is.